src/parse.y

```diff
@@ -1071,6 +1075,34 @@ av_permission:
 	COMMENT
 	;
 
+// policy/global_booleans and policy/global_tunables files
+
+global_conditions_file: // must not allow a comment to be first -> parser conflict
```

Comment:
In default refpolicy both of these files do start with comments.  What is the impact of that on this?  Would we be adding an error message out of the box for anyone trying to run selint on vanilla refpolicy?

Reply:
It's similar to the comment at https://github.com/TresysTechnology/selint/blob/c4cc6e09b8c66fa920c8ebc1e942eac3f554121b/src/parse.y#L993

Leading comment in the file are allowed and accepted, the comment is just about not allowing leading comments in the 
 `global_conditions_file` symbol, cause they are handled in the symbol `selinux_file` via the rule
```
        [...]
        global_conditions_file
	|
	comments global_conditions_file
        [...]
```

Reply:
Thanks, I understand now.